src/bot_methods.js

```diff
@@ -1,314 +1,412 @@
+ * user entered a word as parameter, get the word from the dictionary and send it
+ * @param {Message} message discord message object
+ */
+function handleCommandWord(message) {
+  const word = message.content.split(" ")[1];
+  console.log(`Bot sent word! ${word}`);
+  sendCustomWord(message, word);
+}
+
+/**
+ * Handles the "start" command
+ * starts automatic sending wotd messages and updates word status in the file
+ * sends a new word every 24 hours (as specified in MSEC_PER_DAY constant)
+ * only an admin or moderator can use this command.
+ * @param {Message} message discord message object
+ */
+function handleCommandStart(message) {
+  if (checkIfAdmin(message)) {
+    console.log("Bot started!");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot started! A new word will be posted every 24h from now."
+    );
+
+    sendWordOfTheDay(message, true);
+
+    interval = bot.setInterval(sendWordOfTheDay, MSEC_PER_DAY, message, true);
+  } else {
+    console.log("Bot did not start, user was not a moderator or admin");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot did not start, user must be a moderator or admin to use this command."
+    );
+  }
+}
+
+/**
+ * Handles the "stop" command
+ * stops automatic sending wotd messages
+ * only an admin or moderator can use this command.
+ * @param {Message} message
+ */
+function handleCommandStop(message) {
+  if (checkIfAdmin(message)) {
+    console.log("Bot stopped!");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot stopped! New words will not be posted automatically."
+    );
+
+    bot.clearInterval(interval);
+  } else {
+    console.log("Bot did not stop, user was not a moderator or admin");
+    //logger.logMessage(message);
+
+    message.channel.send(
+      "Bot did not stop, user must be a moderator or admin to use this command."
+    );
+  }
+}
+
+/**
+ * Handles the "help" command
+ * displays information about the bot commands in chat
+ * @param {Message} message
+ */
+function handleCommandHelp(message) {
+  console.log("Bot helped!");
+
+  let help =
+    "Commands for the word of the day bot:\n" +
+    "```" +
+    "$word x  - Searches word x in the dictionary and posts it if available\n" +
+    "$start - Starts automatic sending the word of the day - once a day, from current time (requires moderator permissions)\n" +
+    "$stop  - Stops automatic sending the word of the day (requires moderator permissions)\n" +
+    "$goodbot - Shows your appreciation for the bot\n" +
+    "$goodboy - Woof!\n" +
+    "```";
+
+  message.channel.send(help);
+}
+
+/**
+ * Handles the 'hitme' command
+ * continuously provides words to translate
+ * @param {Message} message
+ */
+function handleChallengeMode(message) {
+  let wordCheck = false;
+  let engWord = null;
+  let estWord = null;
+  while (!wordCheck){ // because some words lack english translations, it loops until a word with a translation is found

```

Comment:
the code is a bit crammed and hard to read. 
add some empty rows, for example before the while block, the filter const, the huge fetchMessages chunk